documentation/asciidoc/accessories/iqaudio/faq.adoc

```diff
@@ -24,24 +23,23 @@ $ sudo alsactl restore -f /home/pi/usecase.state
 === Using external USB devices
 
 If you want to enable 1.2 amp USB support (to allow USB hard disks to power up when
-accessory boards are in use) then you may also want to add the following line to your `/boot/config.txt`
+accessory boards are in use), you may also want to add the following line to your `/boot/config.txt`
 file:
 
 ----
 max_usb_current=1
 ----
 
-Please ensure your Raspberry Pi power supply is able to deliver a minimum of 2.5 amps if you are powering via mini USB and
-enable `max_usb_current`.
+Please ensure your Raspberry Pi power supply is able to deliver a minimum of 2.5 amps if you are powering via mini USB, and enable `max_usb_current`.
 
-=== MPD based audio with volume control
+=== MPD-based audio with volume control
 
-To allow MPD (Music Player Daemon) based audio software to control the audio board’s built in volume the file
-`/etc/mpd.conf` may need to be changed to support the correct ALSA mixer name.
+To allow Music Player Daemon (MPD)-based audio software to control the audio board’s built in volume, the file
+`/etc/mpd.conf` may need to be changed to support the correct AlsaMixer name.
 
-This can be achieved by ensuring the 'Audio Output' section of `/etc/mpd.conf` has the 'mixer_control'
-line. Below is an example for the Texas Instruments based boards (DAC
-PRO/DAC{plus}/DigiAMP{plus})
+This can be achieved by ensuring the 'Audio output' section of `/etc/mpd.conf` has the 'mixer_control'
+line. Below is an example for the Texas Instruments-based boards (DAC
+PRO/DAC{plus}/DigiAMP{plus}):
```

Comment:
Feels a bit uncomfortable with no spaces between the slashes here? (or maybe that's just me)
Also, "Texas Instruments-based boards" feels like a level of technical detail that the users shouldn't need to know?

Reply:
Yeah, the TI bit had me vacillating when I did the first pass. I'll kill it. Slashes/spaces is a style guide thing; no spaces for consistency. 

Not fixing the TI reference for now; another one for a sit-down conversation.